/**:
  ros__parameters:
    cmd_topic_name: 'cmd_vel'

    read_period: 0.01 # seconds
    port_name: '/dev/ttyACM1'
    baud_rate: 57600

    init_time: 3.0 # After the command value is reasonable for this time (seconds), publishing begins
    freeze_check_times: 50 # How many times the cmd value doesn't change to detect freeze

    speed_init: 0.0 # init speed value (m/s)
    steer_init: 0.0

    speed_init_thres: 0.05  # When init, cmd value should be within this threshold for several seconds 
    steer_init_thres: 0.05
    
    speed_max: 5 # m/s 5
    speed_min: -5 # m/s 5 
    steer_max: 0.8 # rad 
    steer_min: -0.8 # rad

    speed_offset: 1500 # This offset should indicate the value at which the car should stop
    speed_ratio: 500 # convert speed from PWM value to m/s  500
    steer_offset: 1500
    steer_ratio: 500 # convert steer from PWM value to radians 500

    debug: False # If True, will print more messages related to publishing.

